kubernetes-archv3/dev-v3-secret.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-dev-v3-env
+  labels:
+    app: dev-v3   
+type: Opaque
+data: 
+    # Kubernetes requires that these environment variables are base 64 encoded 
+    # You can encode a varibale by running  `echo -n 'key' | base64`
+    # You can decode a variable by running  `echo -n 'QWxhZGRpbjpvcGVuIHNlc2FtZQ==' | base64 --decode
+    # Please note that the password here is left for completenes. Real projects should never have passwords commited to renote repos
```

A few minor spelnig notes ;)
```suggestion
    # Please note that the password here is left for completeness. Real projects should never have passwords committed to remote repos.
```